openpype/pipeline/workfile/path_resolving.py

```diff
@@ -187,9 +187,95 @@ def get_workdir(
         project_settings
     )
 
+def get_dotted_extensions(extensions):
```

Comment:
A nommer `get_suffixes` comme dans pathlib je pense

Reply:
I used the same naming than in the current version ([see here](https://github.com/quadproduction/OpenPype/blob/main/openpype/pipeline/workfile/path_resolving.py#L219)). Do I still change ?

Reply:
In my opinion we should be closer to the Python naming convention :
https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.suffixes